Dolphin scripts/Entrance Randomizer/lib/entrance_rando.py

```diff
@@ -67,6 +67,8 @@ class Choice(IntEnum):
 }
 ```"""
 
+connections_left: dict[int, int] = {}
```

Comment:
This is exclusively used for entrance rando generation to temporarily store a state right? If so let's make it private just so it doesn't show up for other modules. (feel free to reword the docstring).

```suggestion
__connections_left: dict[int, int] = {}
"""Used for tracking transitions to randomize during the randomization process."""
```

Reply:
Sure why not.